tools/pipelines/templates/build-npm-package.yml

```diff
@@ -108,6 +108,11 @@ parameters:
   type: string
   default: 'npm ci --unsafe-perm'
 
+- name: additionalBuildSteps
```

Comment:
Can we add some documentation here explaining what this is?

Reply:
Added